src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setAge(18);
+        user.setLogin("Loki");
+        user.setPassword("123456");
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_CorrectUser_Ok() {
+        User registeredUser = registrationService.register(user);
+        Assertions.assertTrue(Storage.people.contains(registeredUser));
+        Assertions.assertTrue(registeredUser.getAge() >= 18);
+        Assertions.assertTrue(registeredUser.getPassword().length() >= 6);
+    }
+
+    @Test
+    void register_LowAge_NotOk() {
+        user.setAge(12);
```

Comment:
варто перевіряти наближені до критичних значення
```suggestion
        user.setAge(17);
```